### PR TITLE
Update test to match the new numeric keys for priority.

### PR DESCRIPTION
**Pull request by bdragon28**
I don't get why stuff is passing for others, it certainly doesn't pass for me without this.

### CODE REVIEW

File: modules/support_ticket/src/Tests/SupportTicketCreationTest.php [hunk 1]
```diff
@@ -49,7 +49,7 @@ function testTicketCreation() {
     $edit = array();
     $edit['title[0][value]'] = $this->randomMachineName(8);
     $edit['body[0][value]'] = $this->randomMachineName(16);
-    $edit['field_priority'] = 'high';
+    $edit['field_priority'] = '120'; // 120 = 'high'
```
Comment (fabianx-ai) [inline modules/support_ticket/src/Tests/SupportTicketCreationTest.php:52]: Should this not be an integer or do we use strings?

Comment (bdragon28) [inline modules/support_ticket/src/Tests/SupportTicketCreationTest.php:52]: I was going by the literal values defined in config/install/field.storage.support_ticket.field_priority.yml -- those are strings.

Comment (jeremyandrews) [inline modules/support_ticket/src/Tests/SupportTicketCreationTest.php:52]: That was almost certainly a mistake on my part. IMO it should be an integer everywhere.

Comment (fabianx-ai) [inline modules/support_ticket/src/Tests/SupportTicketCreationTest.php:52]: Lets follow-up then. I want to have tests passing first.

**Pull request merged**